src/ismeuv.jl

```diff
@@ -0,0 +1,98 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _ismeuv{T<:AbstractFloat}(wave::T, hcol::T, he1col::T, he2col::T, fano::Bool)
+    r = wave/911.75
+    if r > 1
+        return zero(T)
+    end
+    # minexp = -708.39642  from alog((machar(double=double).xmin)) where double = 1b in IDL
+    z = sqrt(r/(1 - r))
+    denom = -expm1(-2*z*π)
+    tauH = ((3.44e-16)*(r^4)*exp(-4*z*atan(1/z))*hcol)/denom
+    r *= 4
+    tauHe2 = zero(T)
+    if r < 1
+        z = sqrt(r/(1 - r))
+        denom = T(4)
+        denom *= -expm1(-2*z*π)
+        tauHe2 = ((3.44e-16)*(r^4)*exp(-4*z*atan(1/z))*he2col)/denom
+    end
+    q  = [2.81, 2.51, 2.45, 2.44]
+    nu = [1.610, 2.795, 3.817, 4.824]
+    fano_gamma = [2.64061e-03, 6.20116e-04, 2.56061e-04, 1.320159e-04]
+    esubi = 4.807317 - (nu.^2).\1
```

Comment:
Looking better, `q`, `fano_gamma`, and `esubi` don't need to be vectors, tuples would work equally good (and are more efficient).  In addition, `esubi` is constant, doesn't depend on input, so in the end you can write
```julia
    q  = (2.81, 2.51, 2.45, 2.44)
    fano_gamma = (2.64061e-03, 6.20116e-04, 2.56061e-04, 1.320159e-04)
    esubi = (4.421529414644497, 4.679309217126802, 4.738680412951545, 4.764345016358231)
```